Makefile

```diff
@@ -205,6 +205,9 @@ gotestsum/%:
 tidy::
 	./scripts/tidy.sh
 
+tidy_check::
```

Comment:
I wonder if we should call this `tidy` and the current `tidy` target `tidy_fix` to be more consistent with other targets?

Reply:
I briefly thought about that, but I was reluctant to change the current behaviour. However since `make tidy` will tell you to run `make tidy_fix`, it's fairly discoverable. I'll go ahead and take that change.